en/docs/administer/key-managers/configure-forgerock-connector.md

```diff
@@ -0,0 +1,288 @@
+# Configure Forgerock as a Key Manager
+
+WSO2 API Manager has the capability to support multiple Key Managers at the same time. So with the use of connectors, it is capable of supporting any authorization server as a Key Manager.
+Additionally it is prepacked with an inbuilt resident Key Manager.
+
+WSO2 API Manager can connect to Forgerock out-of-the-box using the [WSO2 API-M Forgerock Connector](https://github.com/wso2-extensions/apim-km-forgerock).
+
+Follow the instructions given below to configure Forgerock as a third-party Key Manager:
+
+### Configure Forgerock Server to work with APIM
+We assume that you already have a Forgerock installation and follow the steps below to configure the Fogerock Authorization server to work with WSO2 API Manager.
+
+#### Connfigure Oauth2 Provider
+1. Go to the URL where the Forgerock Authorization server is deployed and <br> navigate to 
+   Realms -> Top Level Realm. Top level realm which exists by default is the root realm and the examples and configurations in this guide are based on that.
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-realms.png)
+
+2. Navigate to Root Realm -> Dashboard -> Configure Oauth provider -> Configure Ouath 2.0 and create an oauth provider by filling up the values.
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-createOauthProvider.png)   
+
+3. Navigate to Root Realm -> Service Provider -> Oauth2 Provider-> Advance tab and configure scopes and signing algorithm of the created oauth2 provider as in the image
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-provider-advance-config.png)
+
+4. Navigate to the Dynamic client registration tab and configure it to allow dynamic client registration as follows.
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-configure-dcr.png)
+   
+The other properties of the Oauth2 provider could be configured as per your requirement.   
+
+#### Configure Oauth2 client   
+Once the Oauth2 provider is configured, its time to create a static oauth client, whose keys will be used for dynamic client registration
+
+1. Move to Realm -> Oauth 2.0 -> Applications -> Add client to create a client. Ensure to add the two scopes which are shown in the image while creating the client. These are rquired for dynamic client registration and introspection.
+   
+     ![alt text]({{base_path}}/assets/img/administer/forgerock-add-client.png)
+
+2. Navigate to the above created client under Applications -> client Id of the App -> Core -> Access Token LifeTime and set a long value to Access Token LifeTime to obtain a registration access token which would be used to dynamically register and update clients.
+    
+3. Navigate to the advance tab and confure the client_credential grant type which you need to use to obtain the access token. Configue Password grant type to use the example in step 4.
```

Comment:
- Which step are you referring to in the following sentence?

  "Configue Password grant type to use the example in step 4."

- "Configue" should be "Configure" 
Be mindful of spelling mistakes. Always run the content on a spellchecker before sending a PR.